Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,56 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame: self.view.bounds styleURL: [MGLStyle lightStyleURLWithVersion:9]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.897435, -77.039679) zoomLevel:12 animated:NO];
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    NSURL *url = [[NSURL alloc] initWithString: @"https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpemc0YWlpNzAwcXUyd21ldDV6OWpxMGwifQ.A92RQZpwUgtGtCmdSE4-ow"];
+    if (url) {
+        NSData *data = [[NSData alloc] initWithContentsOfURL:url];
+        MGLShape *feature = [MGLShape shapeWithData:data encoding: NSUTF8StringEncoding error: nil];
```

Comment:
Nit: `error` is of type `NSError **`, which is not an Objective-C object pointer but rather a double-pointer. Use `NULL` instead of `nil` to indicate the nonexistence of a non-object pointer. See [this article](http://nshipster.com/nil/) for more nuance.